cfg/dbconnect_new.php

```diff
@@ -34,9 +31,7 @@
 
 defined( '_PHP_CONGES' ) or die( 'Restricted access' );
 
-#$mysql_serveur="localhost" ;
-#$mysql_user="dbconges" ;
-#$mysql_pass="motdepasse";
-#$mysql_database= "db_conges" ;
-
-
```

Comment:
Je l'ai noté dans le message en haut, le diff s'est vautré, ils sont toujours là. Regarde le [nouveau fichier](https://github.com/Prytoegrian/Libertempo/blob/18aa053c6db0d06c4023dcfb9e8535373700ca9e/cfg/dbconnect_new.php)